.github/workflows/main.yml

```diff
@@ -36,6 +36,7 @@ jobs:
           compiler: gcc
           compiler_version: "11"
           python: 3.7
+          test_generate: ON
           test_render: ON
```

Comment:
In our cloud-based render tests, it looks like "Viewer_BrassAverage.png" ends up as a solid grey square after this changelist!

Reply:
This is an accident of changing some variables to float in the glsl code which should not have be float.
I will correct this.

Reply:
Fix now (current render result)
![Viewer_BrassAverage](https://user-images.githubusercontent.com/14275104/134397048-180c6a61-c4ba-49b7-b7db-5d7d366a603e.png)